{"name": "Проверка https://dev-api.getprime.pro/api/v1/auth - invalid-api-key", "status": "passed", "description": "Невалидная проверка с передачей неверного ключа ", "steps": [{"name": "Проверка https://dev-api.getprime.pro/api/v1/auth на код 401", "status": "passed", "start": 1752136704896, "stop": 1752136705567}, {"name": "Проверка текста message. Ожидали текст 'Вы не авторизованы. Пожалуйста, выполните вход и повторите попытку.' Получили 'Вы не авторизованы. Пожалуйста, выполните вход и повторите попытку.'", "status": "passed", "start": 1752136706170, "stop": 1752136706170}], "start": 1752136704896, "stop": 1752136706170, "uuid": "df46967a-b344-4210-813c-37f15ab942e1", "historyId": "f6aad7747424500f0f55fb65296e0431", "testCaseId": "f6aad7747424500f0f55fb65296e0431", "fullName": "tests.test_api.TestApiGet#test_auth_wrong_key", "labels": [{"name": "tag", "value": "apitest"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApiGet"}, {"name": "host", "value": "Robotek3000"}, {"name": "thread", "value": "12068-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}